opm/grid/common/WellConnections.cpp

```diff
@@ -147,47 +149,37 @@ postProcessPartitioningForWells(std::vector<int>& parts,
                                          return (p1.second > p2.second);
                                      })
                     ->first;
+                // trigger code \todo remove after debugging
```

Comment:
You think we should test this PR with these forced migrations before removing and testing again without?

Reply:
Shoot. That rebase went south.